router/ui-routes.js

```diff
@@ -108,6 +121,51 @@ router.get("/admin/exec", function(req, res) {
         res.render("admin_exec", {
             whiteBackground: true,
             currentExec: exec,
+            roles: execRoles,
+            initialRole: execRoles[0].name,
```

Comment:
Why pass an `initalRole` object instead of just using `roles[0]` in the template?

Reply:
initialRole has been removed